user_docs/en/userGuide.t2t

```diff
@@ -814,6 +814,25 @@ If you want to use these while still being able to use your cursor keys to read
 To toggle single letter navigation on and off for the current document, press NVDA+shift+space.
 %kc:endInclude
 
++++ Other navigation commands +++[OtherNavigationCommands]
+
+In addition to the quick navigation commands listed above, NVDA has commands that have no default keys assigned.
+To use these commands, you first need to assign gestures to them using the appropriate dialog (read [Input Gestures dialog #InputGestures] section for details).
+Here is a list of available commands
+- tab
+- Other commands should be listed here

```

Comment:
Note: Probably best to alphabetize them.

```suggestion
- Article
```

Reply:
Do you mean that I should document all other commands with unassigned gestures?

Reply:
If you could add them in here, that would be most appreciated. I'm sorry for me missing grouping, I have a custom assignment for grouping myself. So Articles, Groupings and Tabs it is.

Reply:
Okay, I've added them. I doubted because it is not entirely logical to document commands with unassigned gestures in a pr that implements quick navigations by tabs.